xbmc/interfaces/python/PythonInvoker.h

```diff
@@ -58,9 +58,20 @@ class CPythonInvoker : public ILanguageInvoker
   std::string m_sourceFile;
   CCriticalSection m_critical;
 
+protected:
+  template<size_t NUMBER>
+  static void initializeModules(const std::array<PythonModule, NUMBER> modules)
```

Comment:
```suggestion
  static void InitializeModules(const std::array<PythonModule, NUMBER>& modules)
```

ref: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#86-methods

Reply:
This part is removed.